Sources/Starknet/Accounts/StarknetAccountProtocol.swift

```diff
@@ -87,6 +87,31 @@ public protocol StarknetAccountProtocol {
 
     /// Execute list of calls as invoke transaction v1
     ///
```

Comment:
```suggestion
```

Reply:
let's refrain from that, as this info is redundant, and we have not added such strings previously (see executev1/v3 above)

Reply:
applicable to other places as well; alternatively if there is a concrete reason to add this, we should do so in all places

Reply:
> Execute list of calls as invoke transaction v1

We have this for every other `executeV1|3`, that is precisely why I added it 😅 , but we could only keep the long version if you think the short one is redundant in this context.

Reply:
every other consecutive, but we do not in the executes above this one.
Both approaches are fine for me, the only thing is for us to be consistent